runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/users/actions/index.ts

```diff
@@ -0,0 +1,7 @@
+import modalsActionsTranslations from './modals';
+
+const usersActionsTranslations = {
+    ...modalsActionsTranslations,
+};
```

Comment:
will be more there?

Reply:
I think so. Actions like editing user should be also placed in here in further users features.